src/ralph/utils.py

```diff
@@ -37,23 +56,66 @@ def import_string(dotted_path):
         ) from err
 
 
-def get_backend_type(backends: BaseModel, backend_name: str):
-    """Returns the backend type from a backend name."""
+def get_backend_type(backend_types: List[BaseModel], backend_name: str):
+    """Return the backend type from a backend name."""
     backend_name = backend_name.upper()
-    for _, backend_type in backends:
+    for backend_type in backend_types:
         if hasattr(backend_type, backend_name):
             return backend_type
     return None
 
 
-def get_backend_instance(backend_type: BaseModel, backend_name: str, options: dict):
-    """Returns the instantiated backend instance given backend-name-prefixed options."""
+def get_backend_class(backend_type: BaseModel, backend_name: str):
+    """Return the backend class given the backend type and backend name."""
+    # Get type name from backend_type class name
+    backend_type_name = backend_type.__class__.__name__[
+        : -len("BackendSettings")
+    ].lower()
+    backend_name = backend_name.lower()
+
+    module = import_module(f"ralph.backends.{backend_type_name}.{backend_name}")
+    for _, class_ in getmembers(module, isclass):
+        if (
+            getattr(class_, "type", None) == backend_type_name
+            and getattr(class_, "name", None) == backend_name
+        ):
+            backend_class = class_
+            break
+
+    if not backend_class:
+        raise BackendException(
+            f'No backend named "{backend_name}" '
+            f'under the backend type "{backend_type_name}"'
+        )
+
+    return backend_class
+
+
+def get_backend_instance(
+    backend_type: BaseModel,
```

Comment:
```suggestion
    backend_types: Enum(BaseModel),
```
Two suggestions: "types" and `Enum`

Reply:
We're only giving it one `backend_type` so not sure about the `types` remark.
Not sure about the Enum also, as we're giving it the reference to the settings class for a type, e.g. `LRSBackendSettings`